extensions/amp-a4a/0.1/crypto-verifier.js

```diff
@@ -49,8 +49,10 @@ export let PublicKeyInfoDef;
  */
 export function importPublicKey(jwk) {
   // WebKit wants this as an ArrayBufferView.
-  return (isWebkit ? utf8Encode(JSON.stringify(jwk)) : Promise.resolve(jwk))
-      .then(encodedJwk => crossCrypto.importKey(
+  const encodedJwk = isWebkit ? utf8Encode(JSON.stringify(jwk)) : jwk;
+  // Despite importKey expecting to return a promise, Safari throws synchronous
```

Comment:
really? I hate it. What's the criteria for deciding whether to throw sync exceptions or return a rejected promise?